src/elasticsearch/aggregates/builders/auto_date_histogram.rs

```diff
@@ -34,9 +34,9 @@ fn auto_date_histogram_agg(
     aggregate_name: &str,
     field: &str,
     buckets: i64,
-    format: Option<default!(&str, NULL)>,
-    minimum_interval: Option<default!(Intervals, NULL)>,
-    missing: Option<default!(&str, NULL)>,
+    format: Option<default!(&str, "NULL")>,
```

Comment:
Didn't we get a raw `NULL` supported for `default!()`?  If we did, I'd prefer to not have this churn.

Reply:
I think I could make a custom keyword: https://docs.rs/syn/1.0.76/syn/macro.custom_keyword.html

Reply:
I think `NULL` is an important enough SQL keyword that it makes sense to try and support it as a free-standing literal.

Reply:
I'm a fangirl of this idea!